src/services/ethereumService.js

```diff
@@ -0,0 +1,251 @@
+import { ethers } from 'ethers'
+import { showToastMsg } from './toastService';
+import TiramisuSavingsClub from '../artifacts/contracts/TiramisuSavingsClub.sol/TiramisuSavingsClub.json'
+const TiramisuAddress = '0xFD60a545a2161895Fe1CDF6bf2B83988dC071318';
+const consoleLoggingEnabled = true;
+
+/**************************************** METAMASK WALLET CALLS *************************************************/
+
+let provider;
+let signer;
+let contract;
+let currentWallet;
+let currentChain;
+let callback;
+
+export function isMetaMaskInstalled(){    
+    const { ethereum } = window;
+    return Boolean(ethereum && ethereum.isMetaMask);
+}
+
+export function isWalletConnected(){
+    if(currentWallet === null || currentWallet === undefined) return false;
```

Comment:
Uninitialized variables in Javascript get undefined by default, not null.

So currentWallet === null will be false even if the value is undefined.

This entire func could be simplified with
```return !!currentWallet```